lib/job/compute-next-run-at.ts

```diff
@@ -80,15 +108,15 @@ export const computeNextRunAt = function(this: Job) {
    */
   const computeFromRepeatAt = () => {
     const lastRun = this.attrs.lastRunAt || new Date();
-    const nextDate = date(repeatAt).valueOf();
+    const nextDate: Date = date(repeatAt);
```

Comment:
before casting "any/unknown" to "Date" I would recommend adding a return type for "date" (not quite sure where date is coming from, if it is from an external library, you could add a wrapper for this function or use a type guard.

Reply:
yeah, good point.
I will address this when I add more type information.